graylog2-server/src/main/java/org/graylog2/rest/RestTools.java

```diff
@@ -120,6 +120,11 @@ public static URI buildExternalUri(@NotNull MultivaluedMap<String, String> httpH
         return uri;
     }
 
+    public static URI buildRelativeExternalUri(@NotNull MultivaluedMap<String, String> httpHeaders, @NotNull URI defaultUri) {
```

Comment:
Can we use this method for the following resources as well?

- DevelopmentIndexHtmlGenerator
- DocumentationResource
- DocumentationBrowserResource